src/System.Data.SqlClient/src/System.Data.SqlClient.builds

```diff
@@ -2,7 +2,12 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <Project Include="System.Data.SqlClient.csproj" />
+    <Project Include="System.Data.SqlClient.csproj">
```

Comment:
Those are specifying that you actually have 2 distinct builds of the library: one for windows and one for linux (actually, unix but we don't have a unix only convention /cc @weshaggard)
This makes it consistent with the package.
The way it was structured before was as if the library only had a single cross-platform build.